fr/integrations/data-analytics/sc-elastic-events.md

```diff
@@ -1,35 +1,264 @@
+chmod 644 /usr/share/centreon-broker/lua/elastic-events-apiv2.lua
+```
+
+## Elastic prerequisites and configuration 
+
+Here are the steps to prepare your 
+
+- An index to store events sent by Centreon. You can create a *centreon_status* 
+index with the following command:
+
+```shell
+curl --user elastic:centreon-es-passwd -X PUT "<elastic_proto>://<elastic_ip>:<elastic_port>/centreon_status" -H 'Content-Type: application/json' \
+ -d '{"mappings":{"properties":{"host":{"type":"keyword"},"service":{"type":"keyword"}, "output":{"type":"text"},"status":{"type":"keyword"},"state":{"type":"keyword"}, "type":{"type":"keyword"},"timestamp":{"type":"date","format":"epoch_second"}}}}'
+```
+
+:warning: If you use a custom **[event format](#event-format)**, you will have to 
+modify the index creation accordingly.
+
+- A user / password with required privileges to POST data to the index.
+
+## Centreon Configuration
+
+### Download Elastic events stream connector
+
+```shell
+wget -O /usr/share/centreon-broker/lua/elastic-events-apiv2.lua https://raw.githubusercontent.com/centreon/centreon-stream-connector-scripts/master/centreon-certified/elastic/elastic-events-apiv2.lua
+chmod 644 /usr/share/centreon-broker/lua/elastic-events-apiv2.lua
+```
+
+## Configuration
+
+To configure your stream connector, you must **head over** the **Configuration --> Poller --> Broker configuration** menu. **Select** the **central-broker-master** configuration (or the appropriate broker configuration if it is a poller or a remote server that will send events) and **click** the **Output tab** when the broker form is displayed.
+
+**Add** a new **generic - stream connector** output and **set** the following fields as follow:
+
+| Field           | Value                                                   |
+| --------------- | ------------------------------------------------------- |
+| Name            | Elastic events                                          |
+| Path            | /usr/share/centreon-broker/lua/elastic-events-apiv2.lua |
+| Filter category | Neb                                                     |
+
+### Add Elastic mandatory parameters
+
+Each stream connector has a set of mandatory parameters. To add them you must **click** on the **+Add a new entry** button located **below** the **filter category** input.
+
+| Type   | Name                    | Value explanation                       | Value exemple                                           |
+| ------ | ----------------------- | --------------------------------------- | ------------------------------------------------------- |
+| string | elastic_url             | the url of the Elastic stack            | `https://elastic-fqdn:9200/`                            |
+| string | elastic_index_status    | Token to use the event collector api    | `centreon_status`                                       |
```

Comment:
elastic_index_status is an index and not a token

need to add elastic_username and elastic_password as mandatory params